Library/Homebrew/cmd/style.rb

```diff
@@ -8,18 +23,93 @@ def style
       ARGV.formulae.map(&:path)
     end
 
+    Homebrew.failed = check_style_and_print(target, :fix => ARGV.flag?("--fix"))
+  end
+
+  # Checks style for a list of files, printing simple Rubocop output.
+  # Returns true if violations were found, false otherwise.
+  def check_style_and_print(files, options = {})
+    check_style_impl(files, :print, options)
+  end
+
+  # Checks style for a list of files, returning results as a RubocopResults
+  # object parsed from its JSON output.
+  def check_style_json(files, options = {})
+    check_style_impl(files, :json, options)
+  end
+
+  def check_style_impl(files, output_type, options = {})
+    fix = options[:fix]
     Homebrew.install_gem_setup_path! "rubocop", "0.39"
 
-    args = [
-      "--format", "simple", "--force-exclusion", "--config",
-      "#{HOMEBREW_LIBRARY}/.rubocop.yml",
+    args = %W[
+      --force-exclusion
+      --config #{HOMEBREW_LIBRARY}/.rubocop.yml
     ]
+    args << "--auto-correct" if ARGV.homebrew_developer? && fix
+    args += files
 
-    args << "--auto-correct" if ARGV.homebrew_developer? && ARGV.flag?("--fix")
+    case output_type
+    when :print
+      system "rubocop", "--format", "simple", *args
```

Comment:
Added the ability to do this, with an `--display-cop-names` option on `brew audit` and `brew style`, so people can try it both ways. Don't know if it should be the default: it's easier to identify, but it gets a little long and noisy-looking.